__tests__/app.test.js

```diff
@@ -39,3 +39,46 @@ describe('GET/api/topics', () => {
       });
   });
 });
+
+describe('GET/api/articles', () => {
+  test('status 200, returns an array of article objects in correct format', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((result) => {
+        const articles = result.body.articles;
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test('results should be sorted by date in descending order', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((results) => {
+        const articles = results.body.articles;
+        expect(articles).toBeSortedBy('created_at', { descending: true });
+      });
+  });
+
+  test('should return status 404 and error message when path is incorrect', () => {
+    return request(app)
+      .get('/api/articled')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toEqual('Not Found');
+      });
```

Comment:
Nice bad path test! Now you've done this once, we know this error handling exists and works for any non-existent endpoints you don't need to do a bad path test again